pkg/handler/v2/machine/machine_test.go

```diff
@@ -321,3 +518,11 @@ func genTestCluster(isControllerReady bool) *kubermaticv1.Cluster {
 func genTestMachine(name, rawProviderSpec string, labels map[string]string, ownerRef []metav1.OwnerReference) *clusterv1alpha1.Machine {
 	return test.GenTestMachine(name, rawProviderSpec, labels, ownerRef)
 }
+
+func genTestMachineDeployment(name, rawProviderSpec string, selector map[string]string, dynamicConfig bool) *clusterv1alpha1.MachineDeployment {
+	return test.GenTestMachineDeployment(name, rawProviderSpec, selector, dynamicConfig)
+}
+
+func boolPtr(i bool) *bool {
```

Comment:
instead of adding our own function you can use the pointer.BoolPtr from "k8s.io/utils/pointer" as its already used in a lot of places.